src/Makefile.am

```diff
@@ -637,6 +636,7 @@ ufsdump_LDADD = \
 	ipc/libipc.la \
 	mgr/libmgr.la \
 	debug/libdebug.la \
+	comm/libminimal.la \
```

Comment:
Disclaimer: I cannot test `ufsdump` changes because that tool does not currently build (`-Wunused-parameter`, `-Wmissing-declarations`, and other problems unrelated to _this_ PR). I think the ufsdump changes in this PR are conceptually correct, but I will remove them if others think that this PR should not touch that rotten tool.

Reply:
FWIW; the reason that old code was being kept around for is now unlikely to occur. So I am okay with leaving it untouched here and dropping entirely later.

Reply:
I removed those ufsdump changes in fc9d761 to avoid arguing about them, under the assumption that ufsdump will be removed in #1013.